bundles/action/org.openhab.action.tinkerforge/META-INF/MANIFEST.MF

```diff
@@ -11,10 +11,12 @@ Bundle-ManifestVersion: 2
 Bundle-Description: This is the TinkerForge action of the open Home Aut
  omation Bus (openHAB)
 Import-Package: org.apache.commons.lang,
- org.openhab.core.scriptengine.action,
+ org.openhab.binding.tinkerforge.ecosystem,
+ org.openhab.binding.tinkerforge.internal.model,
```

Comment:
yes, true … there are two options: a) remove the Action from the Designer due to this flaw or b) accept it for now, fix this issue and work on it afterwards